packages/web/src/components/apps/lib/diff.ts

```diff
@@ -0,0 +1,17 @@
+import * as Diff from 'diff';
```

Comment:
Considered not using a library but the code is non trivial + this is incredibly battle tested.

Picked this one but there were another couple good candidates, notably diff-match-patch from google